requirements/extra.txt

```diff
@@ -7,6 +7,6 @@ torchtext>=0.7
 onnx>=1.7.0
 onnxruntime>=1.3.0
 hydra-core>=1.0
-jsonargparse[signatures]>=3.19.0
+jsonargparse[signatures]>=3.19.3
```

Comment:
why was this change made?

Reply:
This PR requires a bugfix upstream to work:

https://github.com/omni-us/jsonargparse/pull/90

The patch was released with `3.19.3`